libs/react-spa/bff/src/lib/BffContext.tsx

```diff
@@ -2,11 +2,14 @@ import { createContext } from 'react'
 
 import { BffReducerState } from './bff.state'
 
-export interface BffContextType extends BffReducerState {
+export type BffContextType = BffReducerState & {
```

Comment:
converted to type because of BffReducerState is a union type of extended interfaces.